### ISSUE TITLE
WIP builds.

### ISSUE THREAD

**Issue by phly95** [gonetz/GLideN64#885]
On https://github.com/gonetz/GLideN64/releases the last release was posted 8 months ago. Any plans to compile new releases there?

**Comment by vgturtle127** [comment]
@phly95 Not until another release. :laughing:

**Comment by phly95** [comment]
What about a nightly release?

**Comment by phly95** [comment]
Like dolphin

**Comment by vgturtle127** [comment]
@phly95 Well, I don't know if you know about this, but he has been posting test builds in the various issues on this GItHub repository. Those have newer changes and also contain fixes for various encountered issues.

As for nightlies or a buildbot or something, I would be all for it! I test the latest version whenever I can, so it would take a few steps out if I could just download the latest versions.

You can also check EmuCR.com in the meantime, if you are brave enough to try development builds.

**Comment by Nes-player4Life** [comment]
@gonetz Please build a suitable release, for us donors at least.
Thanks.

**Comment by phly95** [comment]
Oh yeah, emucr is pretty awesome.

**Comment by vgturtle127** [comment]
@Nes-player4Life I keep forgetting some people donated for this to become a reality. We probably should do nightly builds, then.

@phly95 Well, you have to be careful sometimes. EmuCR doesn't always get every build correct. But I like to use it for quick tests and just to see new features and stuff.

**Comment by gonetz** [comment]
Last week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2RDgwUjJGZTVnRHM/view?usp=sharing

About plans: I have to fix issues with  new features, fix open regressions issues. Then it can be marked as new release.

**Comment by Nes-player4Life** [comment]
Thanks gonetz!

**Comment by PFCKrutonium** [comment]
@vgturtle127 It shouldn't be too hard to make AppVeyor do builds, and you can have it upload artifacts - @gonetz Would need to look into it though, afaik it's limited to owner/contributors.

**Comment by vgturtle127** [comment]
@PFCKrutonium Oh, can you download AppVeyor builds? I thought it just built them to make sure all the changes didn't cause issues...

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2bjFlLWJuNk14S0E/view?usp=sharing

**Comment by PFCKrutonium** [comment]
@vgturtle127 Indeed you can - OpenRCT2.org/Download in fact uses AppVeyor Artifacts (only ones from a merge though).

**Comment by vgturtle127** [comment]
@PFCKrutonium Oh, cool! I did not know that. Maybe I should get some AppVeyor stuff setup in some of my repos, then.

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2VkZ6ZnE0M0RQZTQ/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Thanks.

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2TTE0VzljR0Z6M3M/view?usp=sharing

you welcome :)

**Comment by nileplumb** [comment]
Thanks gonetz! Had some issues with the emuCR builds on occasion not loading textures etc glad to see builds here now looking forward to next 'stable release'

**Comment by vgturtle127** [comment]
@gonetz Thanks! This is really speeding up my initial testing.

**Comment by DonelBueno** [comment]
Are you going to compile this week's master?

Thanks in advance.

**Comment by gonetz** [comment]
Yes

**Comment by oddMLan** [comment]
Now that you are on it, would you kindly upload your current GlideNUI.lib file so we can compile the releases ourselves without installing gigabytes of QT onto our systems? I have a 32GB SDD and don't ask me how I was able to install Visual Studio 2015 onto it, right now I only have around 3GB of free space left.

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2VUpVbzlXSWYwTlk/view?usp=sharing

current GlideNUI.lib (MSVS 2013 release build):
https://drive.google.com/file/d/0B0YqMPjGo3B2N3djUk9oSkdxTmM/view?usp=sharing

**Comment by vgturtle127** [comment]
@oddMLan I have a 120GB SSD and with Windows and all the applications I normally use installed I have similar issues. It's the main reason I don't have Visual Studio installed right now. :disappointed: 

@gonetz Thanks so much for this!

**Comment by gonetz** [comment]
Master weekly:
https://drive.google.com/file/d/0B0YqMPjGo3B2andkZUdlZVZNdDA/view?usp=sharing

**Comment by gonetz** [comment]
Current Master:
https://drive.google.com/file/d/0B0YqMPjGo3B2UDgybzhtSFpua00/view?usp=sharing

Current GLideNUI.lib:
https://drive.google.com/file/d/0B0YqMPjGo3B2dHEzbG5nU09wSUU/view?usp=sharing

**Comment by gonetz** [comment]
No master builds this week.
Check this for something new:
https://github.com/gonetz/GLideN64/issues/970

**Comment by oddMLan** [comment]
Could you upload GLideNUI.lib for MSVC 2015? I can't use that one since MSVC 2015 complains about incorrect target since that one was compiled for MSVC 2013.

**Comment by gonetz** [comment]
No, I can't for the moment. QT does not support MSVS 2015 yet. I'd like to switch to MSVS 2015, but it is not that easy.

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2SGdKaGFaNWs5N1k/view?usp=sharing

**Comment by theboy181** [comment]
I have not been testing lately so I cant say that the latest build broke HW lighting or not, but I noticed that the flag in MK64 on the title screen is the incorrect color with this build. There are other issues in other games with it enabled as well. Im still testing with my ATI 295X2

**Comment by gonetz** [comment]
@theboy181 I'll check it, thanks.

**Comment by gonetz** [comment]
Tested, no issues with HW lighting found.

**Comment by gonetz** [comment]
Today fixes:
* native resolution fixes. You may use high factors now
* hotfix for depth rendering. Now works with AA

New build:
https://drive.google.com/file/d/0B0YqMPjGo3B2OUpCM1FrRzlsOE0/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Keep up the good work!

**Comment by oddMLan** [comment]
For those who want to compile the latest GLideN64 releases using Visual Studio 2015 without tearing your hair out

current GLideNUI.lib for MSVC 2015
https://drive.google.com/file/d/0BwIRb2FuDWQxMU9TQ2xXbWtZYlk/view?usp=sharing
put under GLideN64\src\GLideNUI\release

updated libpng & zlib (required to compile GLideN64 with MSVC 2015 correctly)
https://drive.google.com/file/d/0BwIRb2FuDWQxT0FYQVB0WlBhTHM/view?usp=sharing
put under GLideN64\src\GLideNHQ\lib

kudos to @Cetacea3691 for their complete guide (#742) which allowed me to compile these libs by myself

**Comment by gonetz** [comment]
So, you managed to compile QT with MSVS 2015. Last time I checked QT sources, their did not include build profile for 2015. How you solved that problem?

**Comment by oddMLan** [comment]
I cloned the latest sources from their git repo at https://code.qt.io/qt/qt5.git  and compiled them using nmake.

``
cd C:\GLideN64\Qt5.6-win32-msvc2015-static
configure -static -release -platform win32-msvc2015 -static-runtime -c++11 -qt-pcre -opengl desktop -no-angle -no-freetype -no-libpng -no-crt -opensource -nomake tools -nomake examples -nomake tests -mp
y
nmake
*
``
Well basically I just followed Cetacea's guide 😁

**Comment by gonetz** [comment]
-platform win32-msvc2015 will work only if that platform is supported.

Please open %QTDir%\qtbase\mkspecs folder and check, does it have win32-msvc2015 subfolder. If it does, please give me qmake.conf from it.

**Comment by oddMLan** [comment]
qtbase\mkspecs
[win32-msvc2015.zip](https://github.com/gonetz/GLideN64/files/263974/win32-msvc2015.zip)

mkspecs\common
[common.zip](https://github.com/gonetz/GLideN64/files/263972/common.zip)

**Comment by gonetz** [comment]
I see, they made common config. Did you replace  -MD and -MDd flags
 by -MT and -MTd before build?

**Comment by oddMLan** [comment]
No, I don't recall doing that. Does that create any issues?
The plugin works perfectly without the those flags.

**Comment by gonetz** [comment]
If you built other code with -MD, it will work ok, but result may not work on other systems. It is not a problem, if you not going to distribute your build.

**Comment by DonelBueno** [comment]
What about last week's build?

**Comment by gonetz** [comment]
I was busy last weekend. May be later. Or next week.

**Comment by Nes-player4Life** [comment]
Take your time Gonetz, RLCF 
(Real Life Comes First)

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2VThaZGxjYi12Yzg/view?usp=sharing

**Comment by Nes-player4Life** [comment]
What do we have for release notes?

**Comment by gonetz** [comment]
Not so much. Check git logs

**Comment by baptiste0602** [comment]
Hello Gonetz,
Could you upload recent build ?
thanks

**Comment by gonetz** [comment]
Current master + software depth buffer render:
https://drive.google.com/file/d/0B0YqMPjGo3B2WU1JdVgtM3FxdFU/view?usp=sharing

**Comment by baptiste0602** [comment]
Thanks

**Comment by gonetz** [comment]
Please test software depth buffer render. It is new functionality for GLideN64, problems are possible. You may run any game with it enabled. Games, where it is surely useful are Zeldas, PD, CBFD.

**Comment by SigmaVirus** [comment]
Thanks Gonetz!

**Comment by gonetz** [comment]
You welcome

**Comment by AmbientMalice** [comment]
@gonetz could you post a new version of GLideN64 with that depth buffer fix for Zelda? I'd like to do some more testing, especially with Snap, but mupen64plus doesn't run it properly.

**Comment by gonetz** [comment]
@AmbientMalice current build of software_depth_render branch:
https://drive.google.com/file/d/0B0YqMPjGo3B2OFdwaGg5RVBScXc/view?usp=sharing

**Comment by SigmaVirus** [comment]
So I was wondering, since these excellent packages only comes with the plugins for Mupen64Plus and Project64 and nothing else, is the GLideN64.custom.ini here on GitHub up to date?

Last change were 2 months ago which seems to be Pokémon Snap related and given the changes made to the plugin since then I can't help to feel my GLideN64.custom.ini that started out as version 3 or something in the beginning which is now up to 11 is out of date.

Had to delete some stuff in it that's not even used anymore I think? Like detectCFB=1 etc.
I can't find that option anymore, neither in the GLideN64.ini of the settings through the emulator.

Anyone that have a updated version of the GLideN64.custom.ini file?

**Comment by gonetz** [comment]
I don't have time to keep GLideN64.custom.ini up to date.
The thing I constantly forget to do about it is to remove ini version check for custom ini.
I need to finish a few things to say: this is pre-release, it will not change much until release, lets polish everything.

**Comment by SigmaVirus** [comment]
OK, thanks for clarification!
Plugin works great BTW, really like the progress.

**Comment by gonetz** [comment]
This week master:
https://drive.google.com/file/d/0B0YqMPjGo3B2aW1GUHNSeTlGSkk/view?usp=sharing

**Comment by theboy181** [comment]
![capture](https://cloud.githubusercontent.com/assets/9299743/16363741/fcef0346-3b88-11e6-8837-a79e6d0db5f9.PNG)


hmm new release has new regression.. ;( (render textrects in native resolution)

**Comment by theboy181** [comment]
Someone able to post an update for me ?

**Comment by AmbientMalice** [comment]
@gonetz Any chance of a new release? I'd like to test the fixed texture cache stuff.

**Comment by gonetz** [comment]
Ok, current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2VEg4aTZvR09BbGc/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Thanks!

**Comment by DonelBueno** [comment]
Could you upload the current master, please?

**Comment by baptiste0602** [comment]
Latest master : https://drive.google.com/file/d/0ByOA3_oOu6O9TkxMMXkwWVRRTmc/view?usp=sharing

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2elZELVRqT1FIMHM/view?usp=sharing

Current GLideNUI.lib:
https://drive.google.com/file/d/0B0YqMPjGo3B2NmZQelpJbm5uM2c/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Thanks!

**Comment by AmbientMalice** [comment]
Is anyone else getting huge performance dips in Perfect Dark in this most recent build? I need to test some older builds to be sure, though...

**Comment by gonetz** [comment]
PD runs as before for me.

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2dW5YYW5aNmpXNDQ/view?usp=sharing

**Comment by gonetz** [comment]
New build:
https://drive.google.com/file/d/0B0YqMPjGo3B2LWtBUDVub00zLUU/view?usp=sharing

New feature added: crop image. Enable it to remove black boarders around image.
Set Crop to Auto and Aspect to Adjust, run Wave Race or AeroGauge in full screen mode...
Note: crop applied only to black boarders from frame buffer. Boarders created by Video Interface will not be removed.

**Comment by theboy181** [comment]
Thanks!! 

Mario Golf has regression where character is missing now.. ;(

**Comment by gonetz** [comment]
Yes, I made a stupid typo, which broke Mario Golf. It will be fixed soon.

**Comment by gonetz** [comment]
Updated:
https://drive.google.com/file/d/0B0YqMPjGo3B2a3ZOcmdEakhuc00/view?usp=sharing

**Comment by jaburgeson** [comment]
Seems there is a regression in the current builds with Episode 1 racer.

Intro flickers and the game shows a black screen.

This only happens with frame buffer emulation turned on.  If it is disabled, the game works correctly.  It used to work with frame buffer emulation turned on.

**Comment by gonetz** [comment]
Recent changes make possible all kind of regressions. I'll check Episode 1 racer, thanks.

**Comment by gonetz** [comment]
Current master+crop feature:
https://drive.google.com/file/d/0B0YqMPjGo3B2Q1JPTTZrWHI4dVk/view?usp=sharing

Fixed most of regression bugs.

**Comment by shinra358** [comment]
On auto, in DK64, you can still see the border but it is flckering in and out rapidly. Also, would like to do each side independantly like reshade or stand alone nestopia just in case there are games where borders are on one side but not the other.

**Comment by olivieryuyu** [comment]
can i get the last master version please?

**Comment by gonetz** [comment]
Current master + crop feature + OSD feature:
https://drive.google.com/file/d/0B0YqMPjGo3B2RHJmU0ZHdVhQUTA/view?usp=sharing

OSD: i made 3 counters: FPS, VI/S, percent. User may select any or all of them. Counters displayed in a user defined corner of the screen. Currently supported only by mupen64plus build.

**Comment by AmbientMalice** [comment]
Gonetz, could you upload a new build with the depth buffer fixes for testing?

**Comment by gonetz** [comment]
Update:
https://drive.google.com/file/d/0B0YqMPjGo3B2UE9JYlBqVzVfeDg/view?usp=sharing

**Comment by Tasosgemah** [comment]
Thanks for keeping us updated.

**Comment by Nes-player4Life** [comment]
Thanks!

**Comment by gonetz** [comment]
Today master:
https://drive.google.com/file/d/0B0YqMPjGo3B2bTlRQTFhR1BHQm8/view?usp=sharing

**Comment by ThallesAps** [comment]
Thank you for your job! But since I'm a totally noob about plugins, can anyone tell me if I need to replace the 1.2 dll with the week's master and place all those files in the Plugin's folder or I can just paste the current dll in that folder, with no other files at all?

**Comment by gonetz** [comment]
@ThallesAps you may just can just paste the current dll in the Plugins folder. The plugin will reset settings to defaults on the first run, so you may want to set them again, at least resolutions.

**Comment by gonetz** [comment]
Today master:
https://drive.google.com/file/d/0B0YqMPjGo3B2cHd3bFg3M2RnYlE/view?usp=sharing

**Comment by lKomus** [comment]
Could you compile the latest master to test the Mischief Makers textures fix please? (I don't know how to compile it myself)

**Comment by gonetz** [comment]
Sure. Today master:
https://drive.google.com/file/d/0B0YqMPjGo3B2ZmVtdHdSbFNDZDA/view?usp=sharing

**Comment by Tasosgemah** [comment]
Thanks again Gonez!

Btw, i never notice any problems with Mischief Makers prior to this master.

**Comment by Gosafer** [comment]
Greetings. For some reason, last master (the compiled one)  is not working for me.
I'm using PJ64 2.3 and I did put the 2 files into the GFX plugin folder and the emulator detected the plugin but, whenever I try to start a game with it, the emulator crashes for no reason.
 The plugin settings seem to work correctly. Maybe I have to change something there? 

It's the first time I try this plugin. Looking forward for some help.

**Comment by lKomus** [comment]
Is there a log file in the folder with Project64.exe and what game and plugins did you use?

**Comment by Gosafer** [comment]
Yes, there's a log file on the logs folder, but it is blank.
 I'm using the default PJ64 plugins:
![captura](https://cloud.githubusercontent.com/assets/22996409/19619886/114e548c-9871-11e6-85cc-d3d8c2dea50a.PNG)

**Comment by lKomus** [comment]
Could I ask for another master compile in order to test the new color clamp fix on Mischief Makers?

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2ZlZQd1FOSHE5aTQ/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Thank you sir. Can we expect an official, polished, form friendly version anytime soon?

**Comment by gonetz** [comment]
Yes. IMO, everything is ready for new release. My goal is to fix all regressions before release. Then I'll tag current state as "Release". I don't have much time to polish it.

**Comment by AmbientMalice** [comment]
@gonetz The default setting should probably be async with software depth buffer, right? It's a good compromise between speed and accuracy, and games that absolutely need sync can have it forced. Games like Hexen, for example.

I think currently FB is disabled by default.

**Comment by gonetz** [comment]
Agreed. Default depth buffer copy should be set to "software"

**Comment by Nes-player4Life** [comment]
Kaspersky internet security reports a gnarly.
![untitled-1](https://cloud.githubusercontent.com/assets/10325597/20042624/8a81fff8-a44b-11e6-9b63-2397cfbab0cf.png)

**Comment by Mario54321** [comment]
Instead of just saying "Aspect ratio" you can specifically put the type of aspect ratio it is (storage aspect ratio, display aspect ratio, pixel aspect ratio)

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2OUJwaUpXV0NRNmM/view?usp=sharing

@Nes-player4Life I checked it with virustotal.

**Comment by budtz** [comment]
The new version gives me black screen with sound running - with AA enabled. Turn that off and i have picture also. Have reverded to version from last week.

**Comment by oddMLan** [comment]
The [New GUI version ](https://github.com/gonetz/GLideN64/issues/1213#issuecomment-260191947) works fine for me with AA. Maybe clear the shader cache and reset settings, that should do it.

**Comment by andykara2003** [comment]
Hi Gonetz - I'm not sure if this is the best/only way to ask you this but will GlideN64 display the 2D HUD elements of Mario 64 (power meter, lives count, Lakitu) smoothly at a higher resolution, say 1280x1024 without looking at all blurry/out of place? Cheers, Andy.

**Comment by gonetz** [comment]
@budtz can you detect GL version, provided by your video card? I made AA related corrections, which can be incompatible with GL below 4.3

@oddMLan GUI build is behind master branch.  It should be used only to test GUI work.

**Comment by gonetz** [comment]
@andykara2003 Is blurry 2D HUD elements a GLideN64 specific issue?  Plugin can't change quality of original low res 2D.

**Comment by budtz** [comment]
Thx for reply Gonetz. How do I check GL version? I have the latest nvidia drivers. Is that enough?

**Comment by gonetz** [comment]
No, but I see the problem.

**Comment by gonetz** [comment]
Update:
https://drive.google.com/file/d/0B0YqMPjGo3B2cVhlRlBOY1FJU1E/view?usp=sharing

**Comment by andykara2003** [comment]
Thanks Gonetz - it's not a GlideN64 specific issue - I was just hoping that the plugin might be able to resolve it but I realise it's not possible now. In that case I guess my only option would be to look for a source where someone has faithfully recreated those elements in higher res. and substitute. Thanks for your work on this plugin :)

**Comment by gonetz** [comment]
@andykara2003 There are plenty of hi-res texture packs for Super Mario 64. GLideN64 supports them. Just place the pack in proper place and enable hi-res packs in settings.

**Comment by andykara2003** [comment]
Thanks very much Gonetz :)

**Comment by gonetz** [comment]
You welcome

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2dzczbnQySEVkYXc/view?usp=sharing

**Comment by Nes-player4Life** [comment]
Thanks for the update. 

I take it that my security just does not like the dll then.

**Comment by gonetz** [comment]
All planned features applied. Release Candidate 1:
https://drive.google.com/file/d/0B0YqMPjGo3B2RWVuUF9DbDBOMm8/view?usp=sharing

**Comment by AmbientMalice** [comment]
@gonetz You're going to have to do something about the texture corruption issue before you release 1.3. Turning of shader cache is the only workaround currently, but that's not ideal.

**Comment by gonetz** [comment]
I need 100% scenario for texture corruption issue

**Comment by shinra358** [comment]
Fullscreening and windowing

changing resolution when textures are loaded

**Comment by budtz** [comment]
Seems OOT subsreen crash is back in RC1 version

**Comment by budtz** [comment]
Hmm seems after i reset settings i cant reproduce it

**Comment by AmbientMalice** [comment]
>I need 100% scenario for texture corruption issue

Rocket: Robot on Wheels seems extremely prone to it. Any translucent texture like glass or water will break when you fullscreen and window.

**Comment by shinra358** [comment]
OoT as well.

**Comment by gonetz** [comment]
> Rocket: Robot on Wheels seems extremely prone to it. Any translucent texture like glass or water will break when you fullscreen and window.

Not for me. 

> OoT as well.

Can't reproduce with it as well.

**Comment by shinra358** [comment]
Well, ill make a video when i finish the walking dead. this was an issue i
noted in another thread. highly reproducable.

**Comment by oddMLan** [comment]
I sure hope that is not a vendor-specific bug. What GFX card do you use, @gonetz ?
I use an AMD R7 205 and I can easily break the transparencies in any game when switching between Windowed and Fullscreen modes.
Also what emulator and OS? I use Project64 2.3 with Windows 10

**Comment by shinra358** [comment]
Im nvidia gtx 980m. Latest dev build of pj64.

**Comment by oddMLan** [comment]
Hm. The transparencies don't break as easily as long as Shader storage is not enabled, but breakage still can happen.

**Comment by shinra358** [comment]
Cant test this version properly because I ran into some issues:

-putting in window mode when textures are loaded causes a crash after items are saved to disk

-minimizing again causes the game to quit and for the emu to have a black screen (so can't change rez during gameplay). Pressing on the black screen will reveal it's back at the menu.


But loading OoT, off the bat, I can see that you can see through the logo.


also, I noticed loading the pack, and doing something else while that happens, will freeze the emu indefinitely graphical wise. But when you do file > close and restart the game, the textures will all be messed up. (found this after I made the video).

https://www.youtube.com/watch?v=YUJbZbHRRGQ

**Comment by gonetz** [comment]
@oddMLan GeForce with Win7 PJ64 2.0, Radeon with Win 10 PJ64 2.2. Several games in a row, switch to full screen and back. Transparency issue never reproduced for me. Heisenbug.

**Comment by weinerschnitzel** [comment]
It takes me about 5 or 7 switches from fullscreen to windowed to trigger texture corrupyion in Turok 3 on my AMD APU.

**Comment by oddMLan** [comment]
Same as @weinerschnitzel

**Comment by Yui12** [comment]
I have this issues as well. Fullscreen and back corrupts graphics.

**Comment by weinerschnitzel** [comment]
Can we get a build to test the fix for #975?

**Comment by AmbientMalice** [comment]
I'd also like a built to test Pokemon Snap, in addition to what @weinerschnitzel said. The plugin seems to be in amazingly better shape than 1.2 now. So many improvements.

**Comment by Gameplayer64** [comment]
Hi there, I am new here and I have an issue with OoT Masterquest. I use an special version (other Link models) but I dont think that is the problem becuase I also have this issue with the normal OoT. You can see the graphic issue here:

![gliden64_the_legend_of_zelda_002](https://cloud.githubusercontent.com/assets/24323935/20850041/b2d89654-b8d9-11e6-8826-d5670061ed54.jpg)
![gliden64_the_legend_of_zelda_000](https://cloud.githubusercontent.com/assets/24323935/20850043/b2e29866-b8d9-11e6-9337-91976b9bf9da.jpg)
![gliden64_the_legend_of_zelda_001](https://cloud.githubusercontent.com/assets/24323935/20850042/b2dc21ac-b8d9-11e6-9e0a-f01ffc787791.jpg)

Thanks for help and sorry for my bad english, I am from Germany

**Comment by gonetz** [comment]
@Gameplayer64 Hacked roms may have all kind of issues. Try to disable frame buffer emulation to get rid of errors.

**Comment by Gameplayer64** [comment]
Thank you for your answer, but I have frame buffer already disabled. And I have the same issue with the normal Ocarina of Time Rom (unhacked).

**Comment by AmbientMalice** [comment]
@Gameplayer64 So you're using the latest WIP build?

**Comment by gonetz** [comment]
New GUI merged to master. Current state:
https://drive.google.com/file/d/0B0YqMPjGo3B2TDJNaldPcnJuZTg/view?usp=sharing

Need to be checked for regressions and texture corruption issues.

**Comment by Gameplayer64** [comment]
Yes, of course i use the newest verison. And also i have this Issue only with Glide N64

**Comment by shinra358** [comment]
So nobody's gonna address the save texture to disk crash, the computer slowdown afterward, or the black screen that happens after you restart the emu and close it down again?

**Comment by weinerschnitzel** [comment]
Relax, the black screen could very well be fixed, as it was deemed related to #975. Do you have tickets for those other issues you are asking about? Ill gladly test them on my end to see if I can reproduce them.

**Comment by shinra358** [comment]
Nope just put them all here a few posts ago. Nobody seemed to acknowledge them so I was under the impression that nobody believes me again :P   All were in a vid I posted above.

**Comment by weinerschnitzel** [comment]
It says the video is unavailable for me.

Just as a general rule, if it does not have a ticket, it is not tracked and people will overlook it.

Like I said I'll gladly test and give feedback. Let's make sure we have tickets to discuss on first.

**Comment by shinra358** [comment]
Great, nobody bothered to tell me, tsk tsk tsk. Okay, I set it to unlisted instead of private. You should be able to see it now.

**Comment by weinerschnitzel** [comment]
@shinra358,

To be fair this is a very long thread that mostly gets used to grab the latest build. So much gets tl;dr'd. 

Please make separate issues for:
1. High resolution texture alignment
  a. make sure you test again with the latest WIP build gonetz posted.
  b. Take a screenshot where there is texture misalignment

2. The crashing from switching full screen and windowed is known with intel cards and is related to #975 
  a. @oddMLan and I have noticed this, but I don't think either of us have created a new ticket specific for intel. Let's make one if it is still reproducible on the latest build
  b. Write the steps as best as you can and include the youtube video second. It helps to reference the issue without watching a 15 minute video.
  c. Also with linking the video, please provide a link to when the bug triggers.

**Comment by shinra358** [comment]
Ok, I'll do that when I get off work.

**Comment by weinerschnitzel** [comment]
Thanks!

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2cWV2UDJqalFrRVU/view?usp=sharing

**Comment by shinra358** [comment]
which particular issue should we be paying attention to that is now fixed in this build?

**Comment by gonetz** [comment]
No particular issue.  The main purpose of this build is to double check that we don't have new regressions.

**Comment by gonetz** [comment]
Update. Fixed #1292. Fixes in GUI. Release Candidate 2:
https://drive.google.com/file/d/0B0YqMPjGo3B2bk51SVo1NjRYMFk/view?usp=sharing

**Comment by gonetz** [comment]
Update. 
*Fixed bug with DePosterization , #1301. The feature did not work since the moment it was merged to master, and only now I got clear signal that something is wrong.
* Gillou68310 made accurate implementation of Golden Eye microcode. Golden Eye microcode is custom modification of standard F3D. See #1300 for details. New implementation brings no visible changes;  the game runs as before. Added some tuning in internal mechanics, to support differences from F3D.

Binary:
https://drive.google.com/file/d/0B0YqMPjGo3B2RDB6cm5VLUZCWXc/view?usp=sharing

**Comment by budtz** [comment]
A small issue in OOT
![img_4486](https://cloud.githubusercontent.com/assets/17853217/21242127/19ced0b0-c313-11e6-86b8-7b5f63c5621b.JPG)

its only here at the hill in the bottom right corner there is a strange texture bug maybe - it follows link as he runs up the hill and then goes away. I dont know if this is related to gliden64 or the emulator

Also. is there a way to fix the lines in the protraits in mario kart?

**Comment by gonetz** [comment]
Problem with road texture in Zelda is plugin related. Decal textures not always work right. 'N64 depth compare' option may help.

Mario cart portraits: use native resolution if this issue bothers you. 2D fixes options also may help.

**Comment by fzurita** [comment]
One interesting thing I have noticed is that using 4xBRz filtering also fixes the Mario kart portraits. I haven't tried other filtering methods.

**Comment by budtz** [comment]
Thx.

The 2d fixes do not help but the 4xBRz filtering does

havnt tried N64 depth compare

**Comment by gonetz** [comment]
Current master:
https://drive.google.com/file/d/0B0YqMPjGo3B2TkVuUF8tTldQN3c/view?usp=sharing

Added ucode for Zelda MM: #848
It brings no visible changes for users.

**Comment by gonetz** [comment]
Update:
https://drive.google.com/file/d/0B0YqMPjGo3B2Z0RmVGE5TkdvZms/view?usp=sharing

Corrected ucode for Zelda MM: #848
Removed ucode for Star Wars Shadows of Empire: #1304
Still there should be no visible changes for users.

**Comment by gonetz** [comment]
Update:
https://drive.google.com/file/d/0B0YqMPjGo3B2U2xYaWRuc1dlUHc/view?usp=sharing

Minor GUI fixes: #1306
More ucode refactoring: #1307

**Comment by gonetz** [comment]
This thread became too long. Closed

**Comment by vgturtle127** [comment]
@gonetz I was wondering when you were going to close it. 😆
